src/CreateCommand.js

```diff
@@ -9,11 +9,12 @@ class CreateCommand {
         let graph = this.dependencyGraph.run();
         let drivers = this.driverLocator.drivers();
 
-        let result = [];
+        let result = {};
 
         graph.forEach((graphElement) => {
-            let driver = drivers[graphElement];
```

Comment:
@troyharvey  Updated this to index on the lowercas driver also, because the driver locator returns a dictionary of drivers, that includes the object name as the key in lowercase format. That way even if you named the file COnTact driver it would still work.